src/content/interop/js-interop/usage.md

```diff
@@ -389,6 +389,47 @@ but can often be elided on and within interop types and on extension members as
 the compiler can tell it is a JS interop type from the representation type and
 on-type.
 
+## Exporting Dart code to JS
```

Comment:
Rather than "exporting Dart code," WYDT about "exporting Dart functions and objects," since those are the two specific techniques you're illustrating in this section?